server/bundler/bin/bundler.js

```diff
@@ -106,4 +106,15 @@ webpack( webpackConfig, function( error, stats ) {
 	assets = utils.getAssets( stats.toJson() );
 
 	fs.writeFileSync( path.join( __dirname, '..', 'assets.json' ), JSON.stringify( assets, null, '\t' ) );
+
+	if ( ! process.env.WEBPACK_OUTPUT_JSON ) {
```

Comment:
It was removed in this PR, but I brought it back behind the flag. I have no idea if we need it at all. I assumed it breaks this patch.